bundles/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/LutronHandlerFactory.java

```diff
@@ -207,13 +209,23 @@ protected synchronized void removeHandler(ThingHandler thingHandler) {
     /**
      * Register a discovery service for an IP bridge handler.
      *
-     * @param bridgeHandler bridge handler for which to register the discovery service
+     * @param thingHandler bridge handler for which to register the discovery service
      */
-    private synchronized LutronDeviceDiscoveryService registerDiscoveryService(IPBridgeHandler bridgeHandler) {
+    private synchronized void registerDiscoveryService(ThingHandler thingHandler) {
```

Comment:
Wouldn't it make more sense to make the parameter a BridgeHandler rather than a ThingHandler?

Reply:
Thanks for your remark. I reverted this part completely.